truss/cli/cli.py

```diff
@@ -64,6 +64,33 @@
     ]
 }
 
+# Note: If you are changing this snippet, please update the example at docs/chains/getting_started.mdx
+HELLO_PY = """import random
```

Comment:
Could we move this into a python file instead of a multiline string?

It has these advantages:
* Undergoes the whole pre-commit checks, i.e. formatting, linting and typing is enforced.
* If code is changed/refactored with tools, it gets updated too (or if missed by manual refactorings, above checks will catch errors).

There is already precedence of this here:
https://github.com/basetenlabs/truss/blob/main/truss-chains/truss_chains/example_chainlet.py
And used as this (the use case there is much more complicated, than what you actually need):
https://github.com/basetenlabs/truss/blob/b8fcdc390b18b8b57cc347467053c06d2c7ab3a3/truss-chains/truss_chains/framework.py#L92

Ideally we can combine all of this, I don't see a reason why the content of `example_chainlet` and `init` could not be the same, and you can feel free to change `example_chainlet` to make it more suitable for your use case.

Reply:
thanks for pointing this out - I went ahead and modified example_chainlet.py to be the example code. I think the advantages make sense. I went ahead and wrote my own helper for loading the example code, since `_example_chainlet_code` wasn't what I needed. LMK WYT